region/norfair/east/Lower Norfair Elevator.json

```
@@ -148,6 +115,14 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 2],
+      "name": "Base",
+      "requires": [
+        {"heatFrames": 60}
```

```suggestion
        {"heatFrames": 55}
```

55 will make it consistent with the Carry Shinecharge strat.